frontend/src/metabase/visualizations/lib/settings/series.js

```diff
@@ -139,9 +152,9 @@ export function seriesSetting({
 
   return {
     ...nestedSettings(settingId, {
+      hidden: true,
+      section: t`Style`,
       objectName: "series",
-      getHidden: ([{ card }], settings, extraProps) =>
```

Comment:
It is possible to access series style settings for a waterfall chart now:
<img width="1713" alt="Screen Shot 2022-10-19 at 6 22 13 PM" src="https://user-images.githubusercontent.com/14301985/196718455-600cbbf5-97b4-4170-8935-d5e8d95c7dd1.png">

Reply:
I mean... it's kinda cool haha